README.md

```diff
@@ -24,6 +24,8 @@ Each incoming DNS query that hits the CoreDNS fanout plugin will be replicated i
   (Cloudflare) will not work.
 
 * `worker-count` is the number of parallel queries per request. By default equals to count of IP list. Use this only for reducing parallel queries per request.
+* `server-count` is the number of DNS servers to be requested. Equals to the number of specified IPs by default. If this parameter is lower than the number of specified IP addresses, servers are randomly selected based on the `load-factor` parameter.
+* `load-factor` - the probability of selecting a server. This is specified in the order of the list of IP addresses and takes values between 1 and 100. By default, all servers have an equal probability of 100.
```

Comment:
What do you think if we define a policy field to be consistent with the forward plugin?

```
fanout  addr1, addr2, adddr3 {
  policy weighted-rand{
    server-count 2
    load-factor 50  50 50
  }
}
```

Reply:
Unfortunately, current version of Caddy (https://github.com/coredns/caddy) does not support nested blocks

Reply:
policy is going to be a separate Caddyfile block that may confuse Coredns users. 

What if we make it flat?

```Caddyfile
example.org {
    fanout . 127.0.0.1:9005 127.0.0.1:9006 127.0.0.1:9007 {
     policy: weighted-random 
     weighted-random-server-count: 2
      weighted-random-load-factor 50 70 100
    }
}
```

Reply:
Done. 

Example:
```
example.org {
    fanout . 127.0.0.1:9005 127.0.0.1:9006 127.0.0.1:9007
    policy weighted-random
    weighted-random-server-count 2
    weighted-random-load-factor 50 70 100
}
```